DMX.Portal.Web.Infrastructure.Provision/Brokers/Clouds/CloudBroker.WebApps.cs

```diff
@@ -19,12 +19,14 @@ public partial class CloudBroker
             var webAppSettings = new Dictionary<string, string>
             {
                 { "ASPNETCORE_ENVIRONMENT", ProjectEnvironment },
+                { "AzureAd:ClientSecret", this.provisionClientSecret },

```

Comment:
Why do we have 2 instances of `AzureAd:ClientSecret`? Lines 22 and 29?

Reply:
@AndrewMcClelland good catch. 

@mimendel ?